DEPS

```diff
@@ -44,12 +44,14 @@ hooks = [
   {
     # Build brave-sync
     'name': 'build_brave_sync',
+    'condition': 'not checkout_android',
```

Comment:
not sure how this is related to rewards

Reply:
It's not related to rewards. @bridiver do you remember why this change was added to android branch?

Reply:
@bridiver won't it cause problems on Android brave-core based? That one I don't think will.

Reply:
yea, this for the extension so it's correct, but not sure why it needs to be added in this PR. Should probably be a separate PR

Reply:
spoke with @SergeyZhukovsky and I understand now so this is ok here